tools/lambda-promtail/Dockerfile

```diff
@@ -1,4 +1,6 @@
-FROM golang:1.24-alpine AS build-image
+ARG GO_VERSION=1.24
```

Comment:
`GO_VERSION` is passed as build arg, see https://github.com/grafana/loki/actions/runs/16257626478/job/45906161654#step:11:189